2017/essence-of-calculus/hebrew/sentence_translations.json

```
@@ -19,91 +19,91 @@
  },
  {
   "input": "The goal here, as the name suggests, is to really get the heart of the subject out in one binge-watchable set.",
-  "translatedText": "המטרה כאן, כפי שהשם מרמז, היא באמת להוציא את לב הנושא בסט אחד שניתן לצפייה בבולמוס.",
+  "translatedText": "המטרה שלי, כפי שהשם מרמז, היא ליצור סדרה אחת קצרה שמכסה את החומר באופן מקיף, ובו בזמן מתעמקת ומגיעה ללב הנושא. ",
   "model": "google_nmt",
   "from_community_srt": "המטרה של הסרטונים האלו היא בעצם לחקור את החדו\"א על בוריה ובכמה סרטונים בלבד.",
-  "n_reviews": 0,
+  "n_reviews": 1,
   "start": 24.3,
   "end": 29.72
  },
  {
   "input": "But with a topic that's as broad as calculus, there's a lot of things that can mean, so here's what I have in mind specifically.",
-  "translatedText": "אבל עם נושא רחב כמו חישוב, יש הרבה דברים שיכולים להיות, אז הנה מה שיש לי בראש ספציפית.",
+  "translatedText": "אבל עם נושא רחב כמו אינפי, למה אנחנו אפילו מתכוונים בלב הנושא? זוהי התוכנית שלי:",
   "model": "google_nmt",
   "from_community_srt": "למרות זאת, בנושא רחב כמו חדו\"א יכולים להיות הרבה דברים לא נעימים. לכן,",
-  "n_reviews": 0,
+  "n_reviews": 1,
   "start": 30.32,
   "end": 36.2
  },
  {
   "input": "Calculus has a lot of rules and formulas which are often presented as things to be memorized.",
-  "translatedText": "לחשבון יש הרבה כללים ונוסחאות שלעתים קרובות מוצגים כדברים שיש לשנן.",
+  "translatedText": "לחדו\"א יש המון נוסחאות וכללים שלעתים קרובות מדי מוצגות כעובדות שיש לשנן.",
   "model": "google_nmt",
   "from_community_srt": "הנה מה שחשבתי: לחדו\"א יש המון כללים ונוסחאות שיכולים להיות בדרך כלל מוצגים בתור דברים שצריך לשנן בעל פה כללי גזירה,",
-  "n_reviews": 0,
+  "n_reviews": 1,
   "start": 36.94,
   "end": 41.94
  },
  {
   "input": "Lots of derivative formulas, the product rule, the chain rule, implicit differentiation, the fact that integrals and derivatives are opposite, Taylor series, just a lot of things like that.",
-  "translatedText": "הרבה נוסחאות נגזרות, כלל המוצר, כלל השרשרת, בידול מרומז, העובדה שהאינטגרלים והנגזרות מנוגדים, סדרת טיילור, פשוט הרבה דברים כאלה.",
+  "translatedText": "הרבה חוקי נגזרות, כלל המכפלה, כלל השרשרת, גזירה סתומה, העובדה שהאינטגרלים והנגזרות מנוגדים זה לזה, סדרות טיילור... פשוט המון דברים כאלה.",
   "model": "google_nmt",
   "from_community_srt": "כלל לייבניץ (לגזירה של מכפלת פונקציות), כלל השרשרת, גזירת אינטגרל מסוים העובדה שאינטגרל ונגזרת הם \"הפכים\" אחד של השני, טורי טיילור יש המון דברים כאלה והמטרה שלי,",
-  "n_reviews": 0,
+  "n_reviews": 1,
   "start": 42.48,
   "end": 52.46
  },
  {
   "input": "And my goal is for you to come away feeling like you could have invented calculus yourself.",
-  "translatedText": "והמטרה שלי היא שתצאי בהרגשה כאילו יכולת להמציא את החשבון בעצמך.",
+  "translatedText": "המטרה שלי עם הסדרה הזאת היא שתצאו בהרגשה שיכולתם לגלות את הנושא הזה בעצמכם.",
```

```suggestion
  "translatedText": "המטרה שלי היא שתצאו בהרגשה שיכולתם לגלות את הנושא הזה בעצמכם.",
```

זה לא הופיע במקורי ואני לא מרגיש שזה מוסיף